hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/OptimizerContext.java

```diff
@@ -225,4 +228,12 @@ public Deque<String> getViewExpansionStack() {
     public Set<PlanObjectKey> getUsedViews() {
         return usedViews;
     }
+
+    public void dump(ILogger logger) {
```

Comment:
Unused method? If removed, then also `QueryPlanner.dump` will be unused.

Reply:
It is very useful during development to see planner state (relsets, costs) but I am not sure if we need be added permanently to logs. This is the same dump we get when planner is stuck, but here it is without exception.